test/components/tree-view.spec.tsx

```diff
@@ -41,7 +54,23 @@ describe('<TreeView />', () => {
         simulate.click(elementToSelect);
 
         return waitForDom(() => expect(elementToSelect).to.have.attr('data-selected', 'true'));
```

Comment:
i agree, but right now i have no definition for how a selected element should look like, so i cannot assert that.